blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -295,8 +296,10 @@ public void run() {
                     TimeUnit.MINUTES.toMillis(savePeriodMins));
         }
 
-        // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
-        new RestartGauge(Metrics.getRegistry(), RollupService.class);
+        if (!SkipInstantiateRestartGauge) {
+            // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
+            new RestartGauge(Metrics.getRegistry(), RollupService.class);
```

Comment:
Got it. I am fine with keeping it here.

FYI. The class is just a wrapper class to the CodeHale metrics API.